src/cluster/base.py

```diff
@@ -7,239 +7,308 @@
 
 
 class Base(Common):
-	"""
```

Comment:
For some reason, indentations were finicky when I was developing...I left it as is to not break it. I'm assuming this fixes everything, since you've tested the entire installation from scratch?

Reply:
This file was `black` formatted to address those finicky indentations in a consistent manner.

It functions as expected in an installation on `scien-hpc`.